packages/react/src/components/UIShell/components/styles/_side-nav.scss

```diff
@@ -426,7 +536,7 @@ div:has(.#{$prefix}--header)
 .#{$prefix}--side-nav__slot {
   display: flex;
   align-items: center;
-  margin: 0 $spacing-05;
+  margin: $spacing-05;
```

Comment:
Is this change intended, the spacing above/below the slot seems like too much? cc @oliviaflory 
<img width="300" height="186" alt="Screenshot 2025-07-14 at 12 58 34 PM" src="https://github.com/user-attachments/assets/3fca4517-ea34-41d3-9ec4-feea12676edd" />